.tx/config

```diff
@@ -9,6 +9,12 @@ file_filter = MapboxNavigation/Resources/<lang>.lproj/Localizable.strings
 source_file = MapboxNavigation/Resources/Base.lproj/Localizable.strings
 source_lang = en
 
+[mapbox-navigation-ios.localizableabbreviations]
+file_filter = MapboxNavigation/Resources/<lang>.lproj/Abbreviations_csv.plist
+source_file = MapboxNavigation/Resources/Base.lproj/Abbreviations_csv.plist
+source_lang = en
+type = PLIST
```

Comment:
Should we override the minimum percentage for this resource to be 100% instead of 80%?